src/index.css

```diff
@@ -62,30 +64,17 @@ h1,
 h2,
 h3,
 h4 {
-  letter-spacing: var(--spacing);
   text-transform: capitalize;
   line-height: 1.25;
   margin-bottom: 0.75rem;
-}
-
-h1 {
-  font-size: 3rem;
-}
-
-h2 {
-  font-size: 2rem;
-}
-
-h3 {
-  font-size: 1.5rem;
-}
-
-h4 {
-  font-size: 0.875rem;
+  font-size: 1.375rem;
+  letter-spacing: 0.2px;
+  color: #121212;
 }
 
 p {
-  margin-bottom: 1.25rem;
+  color: #e8e8e8;
+  font-weight: bold;
 }
 
 @media screen and (min-width: 800px) {
```

Comment:
- [ ] Well done working on the style however you still have some details to work on:
- Kindly use the same font and color as the Zeplin design .

Your design:

<img width="1703" alt="Screenshot 2023-07-20 at 7 28 34 PM" src="https://github.com/aradradev/bookstore/assets/53347966/19ed3351-175e-4d74-98f5-54ecf923364d">